pkg/awsutils/awsutils.go

```diff
@@ -732,47 +734,43 @@ func (cache *EC2InstanceMetadataCache) createENI(useCustomCfg bool, sg []*string
 	return aws.StringValue(result.NetworkInterface.NetworkInterfaceId), nil
 }
 
-func (cache *EC2InstanceMetadataCache) tagENI(eniID string, maxBackoffDelay time.Duration) {
-	// Tag the ENI with "node.k8s.amazonaws.com/instance_id=<instance_id>"
-	tags := []*ec2.Tag{
-		{
-			Key:   aws.String(eniNodeTagKey),
-			Value: aws.String(cache.instanceID),
-		},
+// buildENITags computes the desired AWS Tags for eni
+func (cache *EC2InstanceMetadataCache) buildENITags() map[string]string {
+	tags := map[string]string{
+		eniNodeTagKey: cache.instanceID,
 	}
 
-	// If the CLUSTER_NAME env var is present,
+	// If the clusterName presents,
```

Comment:
nit: If the clusterName presents --> If clusterName is provided

Reply:
make sense